app/controllers/admin/questions_controller.rb

```diff
@@ -4,16 +4,16 @@ class Admin::QuestionsController < Admin::AdminController
 
   def index
     authorize Question
-    # Hide retired questions for now
-    sets = QuestionSet.where(internal_name: %w(loan_criteria loan_post_analysis)).to_a
-    questions = sets.map { |s| top_level_questions(s) }.flatten
-    @json = ActiveModel::Serializer::CollectionSerializer.new(questions, user: current_user).to_json
+    @set_name = params[:set] || "criteria"
```

Comment:
Each question set is now a separate page load instead of loading them all and just hiding the ones not in the current set (!). Much faster.

Reply:
awesome, thank you!